src/Runner.Worker/StepsRunner.cs

```diff
@@ -67,6 +67,11 @@ public async Task RunAsync(IExecutionContext jobContext)
 
                 var step = jobContext.JobSteps.Dequeue();
                 var nextStep = jobContext.JobSteps.Count > 0 ? jobContext.JobSteps.Peek() : null;
+                // TODO: Fix this temporary workaround for Composite Actions
+                if (!String.IsNullOrEmpty(Environment.GetEnvironmentVariable("TESTING_COMPOSITE_ACTIONS_ALPHA")))
```

Comment:
is this required anymore? I thought the scope changes worked-around the issue

Reply:
Tested: This is not needed.